src/Agent.Worker/Release/AgentUtilities.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
```

Comment:
This is a method that formats a specific Release output message. It has exactly one caller.

The caller method's sole purpose is to print these variables. Therefore, the indirection of having this code in a separate class doesn't solve any readability problems.

Also the indirection of a string builder isn't required. The output could be dumped straight to the execution context.

I prefer not moving this into agent.common.util.

I'm guessing the intention for moving this into a separate class is due to testability. As a philosophical side note, I'm skeptical whether the testing this is worth the indirection. I'm fine either way, just posing the question for you to think about.